lib/module.php

```diff
@@ -235,12 +307,24 @@ function solusiovps_CreateAccount(array $params): string
         $solusUser = $userResource->getUserByEmail($params['clientsdetails']['email']);
 
         if (empty($solusUser)) {
-            $solusUserId = $userResource->create([
+            $solusUserData = [
                 'password' => $params['password'],
                 'email' => $params['clientsdetails']['email'],
                 'billing_user_id' => (string) $whmcsUserId,
                 'status' => 'active',
-            ]);
+            ];
+
+            $role = (int) Arr::get($params, 'configoption7');
```

Comment:
Why `configoption7`? Can we use an understandable name here?

Reply:
I would like that as well) I use exactly the same way as before in this project.